hdr/sqlite_modern_cpp.h

```diff
@@ -652,3 +660,5 @@ namespace sqlite {
 		}
 	}
 }
+#undef STR_REF
+#undef U16STR_REF
```

Comment:
Please add the newline at the end of the file.
Also undefining them here seems odd because they are not defined in this file.
This is not a problem for typedefs.